airflow/utils/log/colored_log.py

```diff
@@ -90,8 +90,9 @@ def _color_record_traceback(self, record: LogRecord) -> LogRecord:
 
     def format(self, record: LogRecord) -> str:
         try:
-            record = self._color_record_args(record)
-            record = self._color_record_traceback(record)
+            if self.stream.isatty():
```

Comment:
This is so that `caplog` fixture from pytest doesn't get poluted with escape code.